code/modules/ninja/outfit.dm

```diff
@@ -8,10 +8,10 @@
 	ears = /obj/item/radio/headset
 	shoes = /obj/item/clothing/shoes/space_ninja
 	gloves = /obj/item/clothing/gloves/space_ninja
-	back = /obj/item/tank/jetpack/carbondioxide
-	l_pocket = /obj/item/grenade/plastic/x4
-	r_pocket = /obj/item/tank/internals/emergency_oxygen
-	internals_slot = SLOT_R_STORE
+	l_pocket = /obj/item/grenade/plastic/c4/ninja
+	r_pocket = /obj/item/hypospray/mkii/CMO/combat
```

Comment:
This is new. Any reason for it or just scared ninjas will get btfo too much?

Reply:
iirc tg ninjas can still heal with patches assuming they never removed the fact patches worked through hardsuits
they may have don't quote me on that

on citadel the only way for ninjas to heal without stripping fully is to use their sword on a hypospray to emag it
unfortunately with the loss of the emag act on their sword they kinda need a nudge in the right direction to not get axed by bleed the moment a pellet of buckshot grazes them